lib/howl/commands/app_commands.moon

```diff
@@ -665,39 +665,107 @@ do_search = (search, whole_word) ->
       m = matches[i]
       file_search_hit_to_location(m, search, display_as)
 
-  locations, searcher, project
+  locations, searcher
 
-command.register
-  name: 'project-file-search',
-  description: 'Searches files in the the current project'
-  input: (opts) ->
-    editor = app.editor
-    search = nil
-    whole_word = false
+ask_for_search_directory = (search_query) ->
```

Comment:
Can we rename to `get_search_directory` just for consistency with `get_search_query`?

Reply:
Done